rush.json

```diff
@@ -2237,6 +2237,11 @@
       "projectFolder": "sdk/edgezones/arm-edgezones",
       "versionPolicyName": "management"
     },
+    {
+      "packageName": "@azure/arm-mongocluster",
+      "projectFolder": "sdk/mongocluster/arm-mongocluster",
+      "versionPolicyName": "management"
+    },
```

Comment:
this is not a new library, why we didn't add it last time?

Reply:
there's a rlc pr overwrited the existing config when they resolved conflcts?
https://github.com/Azure/azure-sdk-for-js/pull/30049/files#diff-7cf530b920932d4075d799c0aa86e3757def01fb357fcb27ebc090868ceb62dd